include/aie/Dialect/AIE/IR/AIE.td

```diff
@@ -1403,11 +1403,16 @@ def AIE_ObjectFifoCreateOp: AIE_Op<"objectFifo", [HasParent<"DeviceOp">, Symbol]
         Index:$producerTile,
         Variadic<Index>:$consumerTiles,
         AIE_ObjectFifo_Depth:$elemNumber,
-        TypeAttrOf<AIE_ObjectFifoType>:$elem_type
+        TypeAttrOf<AIE_ObjectFifoType>:$elem_type,
+        OptionalAttr<AIE_DimTupleArrayAttr>:$dimensionsToStream,
+        OptionalAttr<AIE_DimTupleArrayAttr>:$dimensionsFromStream
   );
 
   let assemblyFormat = [{
-    $sym_name `(` $producerTile `,` `{` $consumerTiles `}` `,` $elemNumber `)` attr-dict `:` $elem_type
+    $sym_name `(` $producerTile `,` `{` $consumerTiles `}` `,` $elemNumber 
+      (`,` `toStream` $dimensionsToStream^)?
+      (`,` `fromStream` $dimensionsFromStream^)?
+    `)` attr-dict `:` $elem_type
```

Comment:
Can you put some description of the feature/syntax in the operator's documentation string? And maybe an overview in the PR comments?

Reply:
Yes I will do so and talk about this in the data movement meeting. Just a draft for now